melodic/distribution.yaml

```diff
@@ -9005,6 +9005,24 @@ repositories:
       url: https://github.com/warthog-cpr/warthog.git
       version: kinetic-devel
     status: maintained
+  warthog_desktop:
+    doc:
+      type: git
+      url: https://github.com/warthog-cpr/warthog_desktop.git
+      version: indigo-devel
+    release:
+      packages:
+      - warthog_desktop
+      - warthog_viz
+      tags:
+        release: release/melodic/{package}/{version}
+      url: https://github.com/clearpath-gbp/warthog_desktop-release.git
+      version: 0.0.1-1
```

Comment:
Just noting this is an unusual version if you're using semantic versioning. I don't think it will cause problems with any tooling since it was fine in indigo, but you may consider [bumping to `0.1.0`](https://semver.org/#how-should-i-deal-with-revisions-in-the-0yz-initial-development-phase).